### PR TITLE
Adds the ability to have more than one text box

### PR DESCRIPTION
**Pull request by adkron**
This should lead us to be able to have more than one shape that is more
than text. All the groundwork is in place. One thing that I noticed is
that a name of a shape doesn't have to be unique. Since we have the
unique id I think that having the name be representative of the object
is a good step in helping to debug.

Amos King @adkron <amos@binarynoggin.com>

### CODE REVIEW

**Review by robrieba — APPROVED** [review]
LGTM

**Review by kobsy — APPROVED** [review]
LGTM, too!

File: spec/integration_spec.rb [hunk 1]
```diff
@@ -132,11 +133,41 @@ def pptx(pptx_name)
 
     pptx("text_slide").parts.each do |part_path, expected_part|
       specify "part at #{part_path} has proper content" do
-        expect(content_of(subject, part_path)).to be_equivalent_to(expected_part.content).ignoring_attr_values("Id", "id")
+        expect(content_of(subject, part_path)).to be_equivalent_to(expected_part.content).ignoring_attr_values("Id", "id", "name")
       end
     end
   end
 
+  context "with a slide with two text elemetns" do
```
Comment (ben5516) [inline spec/integration_spec.rb]: spellcheck

> with a slide with two text **elemetns**

Comment (adkron) [reply spec/integration_spec.rb]: Thanks, @ben5516

**Pull request merged**